shell/browser/native_window_views.cc

```diff
@@ -643,6 +643,26 @@ void NativeWindowViews::SetResizable(bool resizable) {
   resizable_ = resizable;
 }
 
+void NativeWindowViews::MoveAbove(const std::string& sourceId) {
+  const content::DesktopMediaID id = content::DesktopMediaID::Parse(sourceId);
+  if (id.type != content::DesktopMediaID::TYPE_WINDOW)
+    return;
+
+#if defined(OS_WIN)
+  const HWND otherWindow = reinterpret_cast<HWND>(id.id);
+  if (!IsWindow(otherWindow))
```

Comment:
I think i'd prefer if this threw a descriptive error of some kind: i think that silently failing is generally not intuitive to users and can lead them to think that our APIs are failing rather than there being an issue on their end.